assets/src/edit-story/components/header/buttons/publish.js

```diff
@@ -65,7 +68,11 @@ function Publish() {
     : null;
 
   const refreshPostEditURL = useRefreshPostEditURL(storyId);
-  const hasFutureDate = Date.now() < Date.parse(date);
+  // Offset the date by one minute to accommodate for network latency.
+  const hasFutureDate = isAfter(
```

Comment:
Should this be wrapped in `useMemo`? This comparison feels like it might be expensive.

Reply:
is it likely that `hasFutureDate` needs to be checked anywhere else, this could be moved to /date and imported.

Reply:
I'll see if I can perhaps just use `isFuture` from date-fns here